### PR TITLE
feature/RMA-92081: checkin on behalf of vishal( grid link button changes)

### PR DESCRIPTION
**Pull request by Shivam3266**

### CODE REVIEW

File: projects/dxc-ngx-cdk/src/lib/dxc-crud-table/dxc-crud-cell-render/dxc-crud-cell-render.component.html [hunk 1]
```diff
@@ -30,7 +38,7 @@
       close</mat-icon>
     <span *ngSwitchDefault>
       <div class="crud-col" *ngIf="formatColumns && formatColumns[columnName]">
-        <span [innerHtml]="rowData[columnName] | formatter:formatColumns[columnName]"></span>        
+        <span [ngClass]="formatColumns[columnName].type == 'button' ? formatColumns[columnName].class : ''" [innerHtml]="rowData[columnName] | formatter:formatColumns[columnName]"></span>
```
Comment (assureclaims) [inline projects/dxc-ngx-cdk/src/lib/dxc-crud-table/dxc-crud-cell-render/dxc-crud-cell-render.component.html:41]: The class should be on button tag

File: projects/dxc-ngx-cdk/src/lib/dxc-crud-table/dxc-crud-cell-render/dxc-crud-cell-render.component.scss [hunk 2]
```diff
@@ -10,3 +10,9 @@
 .red-icon {
   color: red;
 }
+
+.crudLinkBtn
+{
```
Comment (assureclaims) [inline projects/dxc-ngx-cdk/src/lib/dxc-crud-table/dxc-crud-cell-render/dxc-crud-cell-render.component.scss:15]: Css should be on ur app component

**Review by assureclaims — CHANGES_REQUESTED** [review]

File: projects/dxc-ngx-cdk/src/lib/dxc-crud-table/dxc-crud-cell-render/dxc-crud-cell-render.component.html [hunk 3]
```diff
@@ -6,8 +6,11 @@
 </a>
 
 <span class="float-left"
-  *ngIf="(formatColumns && formatColumns[columnName] && formatColumns[columnName].type == 'link'); then linkBlock else contentBlock">
+  *ngIf="(formatColumns && formatColumns[columnName] && formatColumns[columnName].type == 'link'); then linkBlock else elseif1">
 </span>
+<ng-template #elseif1>
```
[the remark this answers is not in the document]
Comment (Shivam3266) [reply projects/dxc-ngx-cdk/src/lib/dxc-crud-table/dxc-crud-cell-render/dxc-crud-cell-render.component.html]: done

File: projects/dxc-ngx-cdk/src/lib/dxc-crud-table/dxc-crud-cell-render/dxc-crud-cell-render.component.html (hunk 1 above)
Comment (Shivam3266) [reply projects/dxc-ngx-cdk/src/lib/dxc-crud-table/dxc-crud-cell-render/dxc-crud-cell-render.component.html]: done

File: projects/dxc-ngx-cdk/src/lib/dxc-crud-table/dxc-crud-cell-render/dxc-crud-cell-render.component.scss (hunk 2 above)
Comment (Shivam3266) [reply projects/dxc-ngx-cdk/src/lib/dxc-crud-table/dxc-crud-cell-render/dxc-crud-cell-render.component.scss:15]: done

**Review by assureclaims — APPROVED** [review]

**Pull request merged**